src/progress.ts

```diff
@@ -116,6 +122,9 @@ export class ProgressController {
       this._logger.log(apiLog, `<= ${this._apiName} failed`);
       await Promise.all(this._cleanups.splice(0).map(cleanup => runCleanup(cleanup)));
       throw e;
+    } finally {
+      this._logRecording = [];
```

Comment:
Well, remove this line from the catch() block now.

Reply:
Done